css/style.css

```diff
@@ -0,0 +1,83 @@
+html, body, {
+    position:fixed;
+    top:0;
+    bottom:0;
+    left:0;
+    right:0;
+
+}
+
+body {
+  text-align: center;
+  color: black;
+  font-size: 80px;
+  text-decoration: none;
+}
+
+header {
+  margin-bottom: 1200px;
+}
+
+h1 {
+  font-family: 'Italiana', serif;
+  font-size: 200px;
+  font-weight: lighter;
+
+}
+
+html {
+    zoom: 1; /* Old IE only */
+    transform: scale(1);
+}
+
+h1 a {
+  text-decoration: none; 
+  color: white;
+  vertical-align: middle;
+}
+
+#bg {
+  background-repeat: no-repeat 50% 50% fixed;
+  background-size: cover;
+}
+
+#aboutimg {
+  margin-left:400px ;
+  margin-right:400px;
+}
+
+nav {
+  margin-top: 80px;
+  font-size: 80px;
+  font-weight: lighter;
+  color: white;
+}
+
+nav a {
+  text-decoration: none;
+  color: white;
+}
+
+h1.bio{
```

Comment:
Combining tags and classes is not necessary and drives up the specificity without need.  🔨